data/en/1_0102.json

```diff
@@ -147,28 +147,48 @@
                 },
                 {
                     "type": "Question",
+                    "id": "reporting-period",
+                    "title": "Reporting period",
                     "questions": [{
+                        "id": "reporting-period-question",
+                        "title": "What are the dates of the period that you will be reporting for?",
                         "type": "DateRange",
+                        "period_limits": {
+                            "minimum": {
+                                "days": 23
+                            },
+                            "maximum": {
+                                "days": 50
+                            }
+                        },
                         "answers": [{
                                 "label": "From",
                                 "type": "Date",
                                 "id": "period-from",
                                 "q_code": "11",
-                                "mandatory": true
+                                "mandatory": true,
+                                "minimum": {
```

Comment:
Do the related test schema also need updating?

Reply:
No.